CMakeLists.txt

```diff
@@ -20,6 +20,7 @@ endforeach()
 message(STATUS "CMake platform: ${CMAKE_SYSTEM}")
 message(STATUS "Build folder: ${AliPhysics_BINARY_DIR}")
 message(STATUS "Source folder: ${AliPhysics_SOURCE_DIR}")
+message(STATUS "Source folder: ${AliPhysics_SOURCE_DIR}/OADB/")
```

Comment:
Nope. Do not ever change the toplevel `CMakeLists.txt`. Change the closest to your tasks's directory.